draft-ietf-rats-corim.md

```diff
@@ -1000,6 +1000,41 @@ measurements for the Target Environment.
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/10
 
+# CoBOM {#sec-cobom}
+
+A BOM object represents the signal for the verifier to activate the listed
+tags. Data contained in a tag MUST NOT be used for appraisal until a BOM which
```

Comment:
Also we must say here that the tag_list comprises a list of tags (CoMID tags and/or CoSWID tags) that were successfully submitted to a verifier in past.

Reply:
not necessarily, they could be shipped contextually.  So the "present" also work :-)